OnlineShop.API/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using OnlineShop.BLL.DTO.RequestDTO.OrdersRequestDTO;
+using OnlineShop.BLL.DTO.ResponseDTO;
+using OnlineShop.BLL.Interfaces;
+using OnlineShop.BLL.Services;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OnlineShop.API.Controllers
+{
+    [Route("api/orders")]
+    [ApiController]
+    public class OrdersController : ControllerBase
+    {
+        private readonly IOrdersService _orderService;
+
+        public OrdersController(IOrdersService orderService)
+        {
+            _orderService = orderService;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<OrdersResponseDTO>>> GetAllOrders()
```

Comment:
you need to throw the token into the methods of the controller, then into the methods of the service and then into the methods of the repositories and into the repositories themselves